src/lib/utils/meta_tables/meta_segments_accurate_table.hpp

```diff
@@ -5,10 +5,10 @@
 namespace hyrise {
 
 /**
- * This is a class for showing information of all stored segments via a meta table.
- * Here, we provide the distinct value count per segment.
- * It is which is expensive to get, as we need to interate over the values of all stored segments.
- * For faster results without that information use MetaSegmentsTable.
+ * This is a class for showing information of all stored segments via a meta table. Here, we provide
+ * - the distinct value count per segment, which is computed if it is not cached, and
+ * - memory usage derived by iterating over all values.
+ * For faster results where this information is less accurate use MetaSegmentsAccurateTable.
```

Comment:
Can you rewrite this sentence? Sounds confusing. Also, don't you mean the inaccurate table?

Reply:
`MetaSegmentsTable provides similar information, but only uses cached distinct value counts and estimates the memory usage by sampling.`?

Reply:
`MetaSegmentsTable provides similar information but more efficiently. It uses only cached distinct value counts and estimates the memory usage by sampling.`
?